app/containers/Forms/FormUpdateUser.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {Row, Col, Form, Button} from 'react-bootstrap';
+import {createFragmentContainer, graphql} from 'react-relay';
+import updateUserMutation from '../../mutations/user/updateUserMutation';
+
+export const FormUpdateUser = props => {
```

Comment:
That's a lot of duplicated code with `FormCreateUser`. There should be a single component.

Reply:
Yea, I'm thinking how to refactor and make it one component

Reply:
Removed this From and using FormEditUser with InputHook